app/controllers/dashboard_controller.rb

```diff
@@ -14,4 +16,16 @@ def index
       end
     end
   end
+
+  def autoyast_profile
+    @suse_register_email = Pillar.value pillar: "suse.register.email"
+    @suse_register_code = Pillar.value pillar: "suse.register.code"
+    @suse_register_server = Pillar.value pillar: "suse.register.server"
+    if @suse_register_email.blank? || @suse_register_code.blank? || @suse_register_server.blank?
+      render status: 503
```

Comment:
Maybe we should add validations for the first page of the setup. We don't want the user to skip these values.